subsys/sal/common/sid_on_dev_cert/sid_on_dev_cert.c

```diff
@@ -633,52 +646,40 @@ sid_error_t sid_on_dev_cert_verify_and_store(void)
 		}
 	}
 
-	result = sid_pal_mfg_store_erase();
+	// init mfg_storage
```

Comment:
minor: this comment seems to bee too detailed (in comparision with others in this file)